mixer/tools/codegen/pkg/interfacegen/generator.go

```diff
@@ -164,28 +166,35 @@ func (g *Generator) getInterfaceGoContent(model *modelgen.Model) ([]byte, error)
 	return imptd, nil
 }
 
-func stringify(protoType modelgen.TypeInfo) string {
-	if protoType.IsMap {
-		return toProtoMap(stringify(*protoType.MapKey), stringify(*protoType.MapValue))
-	}
-	if protoType.IsResourceMessage {
-		return protoType.Name + resourceMsgInstParamSuffix
-	}
-	return "string"
+func trimPackageName(fullName string, pkgName string) string {
+	re := regexp.MustCompile(`(?i)` + pkgName + "\\.")
+	return re.ReplaceAllString(fullName, "")
 }
 
 func (g *Generator) getAugmentedProtoContent(model *modelgen.Model) ([]byte, error) {
 	imports := make([]string, 0)
 
+	var stringify func(protoType modelgen.TypeInfo) string
```

Comment:
worth extracting signature to a non-exported type (`type stringifyFn func(modelgen.TypeInfo) string`) ?